shop/Item.java

```diff
@@ -0,0 +1,41 @@
+package shop;
+
+public class Item {
+    private int ItemID;
+    private float price;
+    private String Category;
+
+    private Supplier supplier;
```

Comment:
```suggestion
    private String Category;
    private Supplier supplier;
```
please be consistent in formatting